src/Symfony/Bridge/Monolog/Logger.php

```diff
@@ -20,7 +19,7 @@
 /**
  * @author Fabien Potencier <fabien@symfony.com>
  */
-class Logger extends BaseLogger implements DebugLoggerInterface, ResetInterface
```

Comment:
This is a breaking change because the `Logger` class won't pass a check like `instanceof Monolog\Logger` anymore.

Reply:
Hey @derrabus, thank you for review. Do you have any suggestion?

Reply:
I think, I've commented my suggestion on both tickets. You cannot fix this class.